.eslintrc

```diff
@@ -1,11 +1,10 @@
 {
   "env": {
```

Comment:
Would be nice to enforce ES6

Reply:
If we do go that route it'd be preferred to use const for the top level variables & let for scoped blocks (might be a change for another PR)

Reply:
ES6 will require transpiling support or bumping up the minimum JS implementation.